src/managers/StorageManager.ts

```diff
@@ -37,12 +37,18 @@ export class StorageManager implements BaseManager<any> {
    *
    * @param id - The name of the JSON file.
    */
-  fetch(id: string) {
+  fetch(id: string): any | void {
     if (this.cache.has(id)) return this.cache.get(id);
     const contentPath = path.join(this._pathName, id + '.json');
     const exists = fs.existsSync(contentPath);
     if (!exists) return;
-    return JSON.parse(fs.readFileSync(contentPath).toString());
+    const content = fs.readFileSync(contentPath).toString();
```

Comment:
This might actually be inaccurate if the file contains whitespaces.
To guarantee this to work right, please `trim` the content before checking it's length.

Additionally, after having parsed the length, it should also verify that the object is of the type `object`.

Reply:
Yeah it won't catch all edge cases. I will fix that